tests/problems/space/test_mixins.py

```diff
@@ -34,8 +34,8 @@ def test_analysis(self, adata_space_rotate: AnnData):
             tgts = set(categories) - set(ref)
             for c in zip(tgts):
                 assert (
-                    adata_ref[adata_ref.obs.batch == c].obsm["spatial_warp"].shape
-                    == adata_ref[adata_ref.obs.batch == c].obsm["spatial"].shape
+                    adata_ref[adata_ref.obs.batch == c].obsm["spatial"].shape
+                    == adata_ref[adata_ref.obs.batch == c].obsm["spatial_norm_warp"].shape
```

Comment:
Think this should be `spatial_warp_norm` (also in other places). Would also test that the normalized coordinates have 0 mean and 1 std.